apps/site/public/googlefaa7f4e155c761f5.html

```
@@ -0,0 +1 @@
+google-site-verification: googlefaa7f4e155c761f5.html
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)